contracts/PrizeStrategyUpkeep.sol

```diff
@@ -70,25 +85,44 @@ contract PrizeStrategyUpkeep is KeeperCompatibleInterface, Ownable {
     /// @param performData Not used in this implementation.
     function performUpkeep(bytes calldata performData) override external {
 
+        uint256 _upkeepLastUpkeepBlockNumber = upkeepLastUpkeepBlockNumber;
+        require(block.number > _upkeepLastUpkeepBlockNumber + upkeepMinimumBlockInterval, "PrizeStrategyUpkeep::minimum block interval not reached");
+
         address[] memory prizePools = prizePoolRegistry.getAddresses();
 
+      
         uint256 batchCounter = upkeepBatchSize; //counter for batch
+
         uint256 poolIndex = 0;
-        
+        uint8 startAwardCounter = 0;
```

Comment:
@asselstine uint8 vs uint256 here?

Reply:
Given gas costs, a limit of 256 is likely fine!

Then again, it's a local variable so using a full word uint256 won't make a difference in terms of gas